doc/source/examples/incompressible-flow/2d-lid‐driven-cavity-flow/lid‐driven-cavity-flow.rst

```diff
@@ -0,0 +1,306 @@
+==================================
+2D Lid-Driven Cavity Flow
+==================================
+
+This is the first Lethe example. It showcases a classical fluid mechanics problem, the lid-driven cavity. This example also introduces the concept of parameter files to parametrize Lethe simulations.
+
+Features used in this example
+----------------------------------
+- Solver : ``gls_navier_stokes_2d``, ``gd_navier_stokes_2d`` (with Q2-Q1)
+- Steady-state problem
+- Setting initial conditions
+- Adjoint time-stepping for steady-state problems
+
+
+Location of the examples
+------------------------
+- Base case (:math:`Re=400`) : ``/examples/incompressible_flows/cavity.prm``
+- :math:`Re=7500` : ``/examples/incompressible_flows/Reynolds_7500/cavity.prm``
+
+
+Description of the case
+-----------------------
+
+The lid-driven cavity is a classical fluid dynamics problem . It consists in the flow in an enclosed cavity, where one of the wall (in this example, the top wall) is put in motion at a constant tangential velocity. This case is often used to benchmark the capacity of CFD software because of the simplicity of it's geometry and the availabily of multiple reference results over a large range of Reynolds number. In this example, we investigate this problem in 2D, but simulating this case in 3D would require only a few changes. The geometry and boundary conditions are illustrated in the following figure:
```

Comment:
```suggestion
The lid-driven cavity is a classical fluid dynamics problem . It consists in the flow in an enclosed cavity, where one of the walls (in this example, the top wall) is put in motion at a constant tangential velocity. This case is often used to benchmark the capacity of CFD software because of the simplicity of its geometry and the availability of multiple reference results over a large range of Reynolds numbers. In this example, we investigate this problem in 2D, but simulating this case in 3D would require only a few changes. The geometry and boundary conditions are illustrated in the following figure:
```